_linux/git.xml

```diff
@@ -0,0 +1,6 @@
+<application>
```

Comment:
Hi, Please remove this file from this pull request and other _linux files since these are specific to your fork. Do this by first copying these files on your local computer, then remove them from git, and then copy them back but not commit them back into the branch. 

Unfortutely we cannot currently offer a better why for you to do this. If you have local git config files you want to track, a possible workaround is to store these in a separate repository of your own